image/image.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/image/image.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Images</h1>
+    <h3>Responsive image</h3>
+    <div class="img-container">
+      <img
+        class="responsive-img"
+        src="https://picsum.photos/700/350"
+        alt="demo-image"
+      />
+    </div>
+    <h3>Round image</h3>
+    <div style="height: 15rem; width: 15rem" class="img-container">
```

Comment:
Better to avoid Inline styles

Reply:
Yes, I will remove these in the refactor.